### DISCUSSION TITLE
新版本 pikpak分享报错不能读取文件

### DISCUSSION THREAD

**Discussion by lydgly** [AlistGo/alist#6510] [General]
failed get objs: failed to list objs: Sorry, sharing is not available in the current region

**Comment by li-peifeng** [comment]
写了所在地不提供，挂代理吧

**Comment by mayziran** [comment]
额，我最近也开始这样，之前好的，pikpak正常，就pikpak分享这样

**Comment by a86827531** [comment]
一样的问题。。挂代理也不行
